src/Message/AbstractMessage.php

```diff
@@ -124,6 +124,22 @@
     }
 
     /**
+     * Get message references (from headers).
+     *
+     * @return array
+     */
+    final public function getReferences()
+    {
+        $references = $this->getHeaders()->get('references');
+
+        if (\is_array($references)) {
+            return \array_unique($references);
```

Comment:
I don't like it very much to alter a header content. Emailing is a brittle system full of exceptions: even though it seems harmless, I would prefer to let the user do it if needed.